style.css

```diff
@@ -84,3 +84,138 @@ ul {
 .next_button {
   margin-top: 50px;
 }
+
+/* Works Section */
+
+.title {
+  margin: 30px 24px 20px 24px;
+  font-weight: bold;
+  font-size: 50px;
+  text-decoration: none;
+  text-align: center;
+}
+
+hr {
+  height: 6px;
+  background-color: #36b37e;
+  border-radius: 24px;
+  width: 60px;
+  border: none;
+  margin-left: 210px;
+  margin-top: 30px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto 30px 20px 55px;
+}
+
+.img_placeholder {
+  width: 100%;
+  height: 350px;
+}
+
+.project_info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  border-radius: 4px;
+  background-color: white;
+  width: 100%;
+  height: 230px;
+  border: 2px #ebf0ee solid;
+}
+
+.project_name {
+  color: #3a4a42;
+  text-align: center;
+  font-size: 25px;
+  cursor: pointer;
+}
+
+.languages_used li {
+  border-radius: 4px;
+  padding: 8px 12px 8px 12px;
+  margin: auto 10px auto 10px;
+  background-color: #ebf0ee;
+  color: #3a4a42;
+  font-size: 12px;
+  font-weight: bold;
+  cursor: pointer;
+}
+
+.see_project {
+  text-decoration: none;
+  background-color: #36b37f;
+  color: #fff;
+  padding: 12px 12px 12px 12px;
+  border-radius: 4px;
+}
+
+.cards {
+  display: grid;
+  row-gap: 10px;
+  grid-template-rows: repeat(5, 600px);
+  grid-template-columns: repeat(1, 450px);
+  margin-top: 60px;
+}
+
+/* About Me Section */
+
+.detailed_info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  background-image: url("./images/illustration-aboutme-mobile.svg"), url("./images/Mask_Group.svg");
+  background-position: left bottom, right top;
+  background-repeat: no-repeat, no-repeat;
+  height: 450px;
+  width: 450px;
+  margin-bottom: 20px;
+}
+
+.card2_content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
### OPTIONAL
- [ ] For DRY css you should combine duplicate properties. Example, you can create class `.d-flex { display: flex, flex-direction:column, align-items: center}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)